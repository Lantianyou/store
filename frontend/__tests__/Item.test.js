import ItemComponent from '../components/Item'
import { shallow, mount } from 'enzyme'
import formatMoney from '../lib/formatMoney'

const fakeItem = {
  id: 'id',
  title: 'title',
  price: 5000,
  descriotion: 'lorem',
  image: 'dog.jpg',
  largeImage: 'largeDog.jpg'
}

describe('<Item />', () => {
  it('d', () => {
    const price = '$50.43'
    expect(price).toMatchSnapshot()
  })
  // it('pricetag and title', () => {
  //   const wrapper = shallow(<ItemComponent item={fakeItem} />)
  //   const PriceTag = wrapper.find('PriceTag')
  //   expect(PriceTag.children().text()).toBe(formatMoney(fakeItem.price))
  //   expect(wrapper.find('Title a').text()).toBe(fakeItem.title)
  // })
  // it('img', () => {
  //   const wrapper = shallow(<ItemComponent item={fakeItem} />)
  //   const img = wrapper.find('img')
  //   expect(img.props().alt).toBe(fakeItem.title)
  // })
  // it('button', () => {
  //   const wrapper = shallow(<ItemComponent item={fakeItem} />)
  //   const buttonList = wrapper.find('.buttonList')
  //   expect(buttonList.children()).toHaveLength(3)
  //   expect(buttonList.find('Link').exists()).toBe(true)
  // })
})
