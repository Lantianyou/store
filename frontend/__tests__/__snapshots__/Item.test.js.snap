// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Item /> d 1`] = `"$50.43"`;
